charts/atlantis/templates/statefulset.yaml

```diff
@@ -138,6 +138,12 @@ spec:
         - name: {{ .Chart.Name }}
```

Comment:
If your intention is to revert #203 then why overwrite the file? Why not simply revert the pr ?

cc: @nxf5025 since you are the original pr author of #203

Reply:
Hey sorry missed the previous notification for this. I agree with @kpettijohn on reverting just the gitconfig changes and not the .netrc as that's confirmed working properly

Reply:
@kpettijohn friendly ping

@mattjamesaus see this thread. There is still changes that need to be addressed. Please feel free to propose a new pr if you can fix these issues with the feedback